extensions/http-curl/tests/CMakeLists.txt

```diff
@@ -40,8 +40,8 @@ FOREACH(testfile ${CURL_UNIT_TESTS})
 	target_wholearchive_library(${testfilename} minifi-civet-extensions)
 	target_wholearchive_library(${testfilename} minifi-standard-processors)
 
-  	MATH(EXPR CURL_INT_TEST_COUNT "${CURL_INT_TEST_COUNT}+1")
-#	 add_test(NAME "${testfilename}" COMMAND "${testfilename}" WORKING_DIRECTORY ${TEST_DIR})
+ 	MATH(EXPR CURL_INT_TEST_COUNT "${CURL_INT_TEST_COUNT}+1")
```

Comment:
What was the goal here? Previously we had 2 spaces + 1 tab, now 1 space + 1 tab. Both are incorrect IMO. The file seems to be using tabs for indentation.

Reply:
I removed the leading space which made the row alignment correct in my editor and I did not notice there was another leading space left. Replaced all leading spaces with tabs in 7d2ce7d94bc3ce060822f6cf3bf0f770c7b36bde